src/cli.rs

```diff
@@ -129,6 +129,10 @@ pub struct Opt {
     #[structopt(env = "GITHUB_APP_INSTALL_ID")]
     pub github_app_install_id: String,
 
+    /// Heroku API token
+    #[structopt(env = "EVE_HEROKU_TOKEN")]
```

Comment:
I'm not certain, but it seemed like Heroku didn't like it when I called this `HEROKU_TOKEN`.